fmn/lib/db.py

```diff
@@ -20,50 +20,68 @@
 """
 from __future__ import print_function, unicode_literals
 
-import sys
+import argparse
```

Comment:
Yeah, click looks good, but for this (which is only meant to create the database) isn't going to be a complex CLI. If it starts to grow, I'll reconsider using click!